Formula/jr.rb

```diff
@@ -0,0 +1,21 @@
+class Jr < Formula
+  desc "CLI program that helps you to create quality random data for your applications"
+  homepage "https://jrnd.io/"
+  url "https://github.com/ugol/jr/archive/refs/tags/v0.3.3.tar.gz"
+  sha256 "fa60365c0ca7b5ff70ef357ff362c7da069aa07a5daa8303f0af04ae75d04f67"
+  license "MIT"
+  head "https://github.com/ugol/jr.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    bin.install Dir["build/*"]
+    prefix.install "config/jrconfig.json"
+    prefix.install "templates"
+  end
+
+  test do
+    assert_match "net_device", shell_output("JR_HOME=$(brew --prefix jr) jr template list").strip
```

Comment:
Why does it need to set `JR_HOME`?

Reply:
The tool loads a set of templates in tpl format, these templates are under JR_HOME. I can test the `prefix.install "templates"` in this way. Is it a bad practice for a test?

Reply:
It means that every user will need to do that too right?

Reply:
yep, or they can create their own templates and use a different JR_HOME.

Reply:
Then you should probably use `write_env_script` to set a reasonable default.